src/ConfigEditor/Auth/common/SecretInput.tsx

```diff
@@ -17,6 +17,9 @@ export type Props = React.ComponentProps<typeof Input> & {
 export const CONFIGURED_TEXT = 'configured';
 export const RESET_BUTTON_TEXT = 'Reset';
 
+/**
+ * @deprecated use the component from @grafana/ui instead. Available starting from @grafana/ui@9.2
+ */
```

Comment:
@cletter7 do we still need to support Grafana 8 anywhere or can these components be removed from here?

Reply:
I think we can remove them. In enterprise datasources we now support grafana 9.5+